src/onegov/form/parser/form.py

```diff
@@ -185,6 +186,18 @@ def handle_field(
             description=field.field_help
         )
 
+    elif field.type == 'video_url':
+        # FIXME: I don't understand mypy complaints here
```

Comment:
That's exactly because it's missing from the `ParsedField` tagged union, this unreachable error goes away once you fix that.

Reply:
Nice!